{"name": "test_03_querySecOrder_报文内容中为空为空字符串时接口给出正确的错误提示", "status": "passed", "description": "\n    报文内容中为空为空字符串时接口给出正确的错误提示\n    :return:\n    ", "attachments": [{"name": "log", "source": "35360f74-0c5b-4eeb-8806-b9e893c0c0dd-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "7a76ba4f-cc0e-48df-997f-eb89dce9ca7c-attachment.txt", "type": "text/plain"}], "start": 1603871049118, "stop": 1603871049204, "uuid": "f1123ae9-50bc-41ef-a710-8b4bd65d14cd", "historyId": "b8ca314d28e0d79f55b994f63049a664", "testCaseId": "8e24a005d925ca72a10a6915a9f9f4fe", "fullName": "testCase.zzj.test_OrderInquiry#test_03_querySecOrder_报文内容中为空为空字符串时接口给出正确的错误提示", "labels": [{"name": "parentSuite", "value": "testCase.zzj"}, {"name": "suite", "value": "test_OrderInquiry"}, {"name": "host", "value": "LAPTOP-8APLEE6F"}, {"name": "thread", "value": "19384-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCase.zzj.test_OrderInquiry"}]}